### ISSUE TITLE
Level 0 Bug - Can't Respec Anyone Other than Bugged Char

### ISSUE THREAD

**Issue by Sikosh1999** [BarleyFlour/RespecMod#45]
Hey BarleyFlour!

First of all thanks for all your work on this mod!. I ran into an issue with a level 0 char, Sosiel, when I recruited him at the camp. 

Today I grabbed the latest update with the level 0 fix, but now he's the only one showing in the respec list despite reloads and restarts.

Is there anyway to fix it or is my save busted? 

![leevel0](https://user-images.githubusercontent.com/90407441/132720721-891e8e95-a433-4a7d-96f3-5499786c3b7c.png)

Thanks!

**Comment by vikigenius** [comment]
Yep I encountered the exact same bug just now, I think the latest bugfix release is the issue,

**Comment by Sikosh1999** [comment]
I'm at Drezen now, and already failed his quest, so think he left for this playthrough :(

Still, I'm stuck not being able to respec anyone else, which is less than ideal!

**Comment by FRRago** [comment]
Try to uninstall the Unit mod manager, then reload the game.
I got that 0-level bug with one of the Mongrells

**Comment by scolemanselby** [comment]
Currently having this bug with Regill after recruiting him at Lost Chapel (because I decided not to go help the Hellknights earlier). Uninstalling Unity Mod Manager and reloading the game does not solve this bug. Losing a couple hours backtracking to when I recruited him.
